src/main/groovy/org/broadinstitute/orsp/consent/ConsentResource.groovy

```diff
@@ -22,6 +22,12 @@ class ConsentResource {
     Timestamp sortDate
     String translatedUseRestriction
     DataUseDTO dataUse
+    String geneticStudiesOnly
+    Boolean publicationResults
+    Boolean genomicResults
+    String genomicSummaryResults
+    Boolean collaborationInvestigators
```

Comment:
The consent resource object is what is used to create the JSON object that is posted to https://consent.dsde-prod.broadinstitute.org/#!/Consent/post_api_consent. These new fields are not part of that schema:
```
{
  "consentId": "string",
  "requiresManualReview": true,
  "dataUseLetter": "string",
  "useRestriction": "string",
  "dataUse": "string",
  "name": "string",
  "dulName": "string",
  "createDate": "2019-12-19",
  "lastUpdate": "2019-12-19",
  "sortDate": "2019-12-19",
  "translatedUseRestriction": "2019-12-19",
  "lastElectionStatus": "string",
  "lastElectionArchived": true,
  "groupName": "string",
  "updateStatus": true,
  "dacId": 0
}
```
Not sure if this has been tested against dev, but my suspicion is that it will might to validate and hence, fail to export. I would be happy to be wrong here.

Reply:
@rushtong you are right. It fails against DUOS dev ( Unable to post consent [CG-4010]: Bad Request). I think it's needed to create an story on DUOS to handle this new fields.
Should we put on hold this stories?
Thanks!